tests/framework/extensions/clusters/clusters.go

```diff
@@ -1091,40 +1096,46 @@ func logClusterInfoWithChanges(clusterID, clusterInfo string, summary summary.Su
 	return clusterInfo
 }
 
-// WatchAndWaitForCluster is function that waits for a cluster to go unactive before checking its active state.
-func WatchAndWaitForCluster(client *rancher.Client, steveID string) error {
-	var clusterResp *v1.SteveAPIObject
-	err := kwait.Poll(500*time.Millisecond, 2*time.Minute, func() (done bool, err error) {
-		clusterResp, err = client.Steve.SteveType(ProvisioningSteveResourceType).ByID(steveID)
+func WatchAndWaitForCluster(client *rancher.Client, ns string, clusterName string) error {
```

Comment:
Why did you change this? Why not just get the Steve ID how it was previously written? I just don't see the point on reverting this? I changed it for a reason, less parameters.

Reply:
That's an accident with rebasing, I didn't touch this function at all, outside of log message.